src/main/java/org/candlepin/model/ConsumerCurator.java

```diff
@@ -1212,6 +1212,46 @@ WHERE c.uuid IN (:uuids)
         return existingUuids;
     }
 
+    /**
+     * Determines if any of the provided {@link Consumer} UUIDs do not exist or do not belong to the provided
+     * {@link Owner}.
+     *
+     * @param consumerUuids
+     *  the consumer UUIDs to check
+     *
+     * @param ownerKey
+     *  the key of the owner that should own the consumers
+     *
+     * @return all of the provided consumer UUIDs that do not exist or do not belong to the provided owner
+     */
+    public Set<String> getNonExistentConsumerUuids(Collection<String> consumerUuids, String ownerKey) {
```

Comment:
nit: owner key should be first in the parameter order, as it has higher impact on the output.

Reply:
Updated